components/launcher-client/Cargo.toml

```diff
@@ -11,8 +11,9 @@ env_logger = "*"
 habitat_core = { path = "../core" }
 habitat-launcher-protocol = { path = "../launcher-protocol" }
 habitat_common = { path = "../common" }
-ipc-channel = { git = "https://github.com/habitat-sh/ipc-channel", branch = "hbt-windows" }
+ipc-channel = { version = "0.16.0" }
```

Comment:
I actually don't quite understand why we do that in most of our dependencies. Is it to enable some kind of dependency upgrade automation ? I generally version my dependencies at-least with a major.minor version to prevent unexpected subtle behavior changes from creeping in.

Reply:
Yeah if this were a public facing crate we would probably do that. We also do it if we know a fuuture version breaks us and it is too time intensive to fix in the near future. However we like to keep it "*" where we can so that dependabot and `cargo update` can update to latest. We try to keep everything at latest. We version control the cargo.lock so ci builds do not try to alter the versions.

Reply:
Got it, will make this change